### PR TITLE
Update docblock

### PR DESCRIPTION
**Pull request by ceeram**

### CODE REVIEW

**Comment by ADmad** [comment]
The `Cache` class also defines the same property which is generating error.

File: src/Core/StaticConfigTrait.php [hunk 1]
```diff
@@ -36,6 +34,14 @@
     protected static $_config = [];
 
     /**
+     * An array mapping url schemes to fully qualified caching engine
```
Comment (ADmad) [inline src/Core/StaticConfigTrait.php]: It's not just for `caching engine`. The mapping can be potentially used any class using this trait.

**Comment by AD7six** [comment]
I don't think this is a good idea - if the trait declares the property, then no class using the trait can declare it differently.

**Comment by ADmad** [comment]
@AD7six Yup.

Closing for reason above.

**Comment by ceeram** [comment]
Reopening to push amended commit

**Pull request merged**